.github/workflows/release-source.yml

```diff
@@ -123,6 +124,11 @@ jobs:
             # put in their own folder.
             FOLDER="${{ env.FOLDER_BRANCHES }}/${BRANCH}"
             TRIGGER_TYPE="new-branch"
+
+            # Generate changelog
+            revdate=$(git log -1 --pretty=format:"%ci")
+            last_week=$(date -d "$revdate -7days" +"%Y-%m-%d %H:%M")
+            git log --after="${last_week}" --pretty=fuller > .changelog
```

Comment:
Totally out of scope of this PR, but maybe you agree:

I was looking at https://cdn.openttd.org/openttd-nightlies/2024/20241224-master-gf74b0cf984/changelog.md, and realised how very ugly it looks. Shouldn't we just use `git log --oneline` instead?

Also, what is related, maybe we should prefix the changelog with a header now it is markdown. Something like:

```
echo "## Changes since ${last_week} UTC" > .changelog
echo "" >> .changelog
```

(and then ofc change the `git log` line to use `>> .changelog`.). Not sure what a good text for the header is. Just a random thought.

Reply:
`--oneline` would be a big improvement.